src/Components/ContentDelivery/ContentDelivery.module.scss

```diff
@@ -11,50 +11,57 @@
   flex-direction: column;
   align-items: center;
   justify-content: center; 
-  margin: 1em; 
+  margin: 1em 1em 0;
 }
 
 .videoWrapper {
-  @include size(60em, 34em);
+  @include size(42em, 21.4em);
+  display: flex;
+  justify-content: center;
 }
 
 .videoPlayer {
   @include borderAndShadow(none, 1em, 0px 0px 20px 2px $midBlue);
   background-color: $midBlue; 
+  background: radial-gradient($midBlue, #7987a1ab);
+  box-shadow: 0px 0px 20px $darkBlue, inset 0px 0px 5px rgba(0, 0, 0, 0.239);
```

Comment:
Love the consistent box shadows throughout the site; really gives it a unified feel

Reply:
note for the future, these shadows would be a great addition to the `variables` file.